glide.yaml

```diff
@@ -12,6 +12,10 @@ import:
   version: da575b14ac9ceb113f18f1c1cab75be256460d64
   subpackages:
   - design
+- package: github.com/fabric8-services/fabric8-toggles-service
```

Comment:
Is this needed?

Reply:
no it's not, and I thought I had removed it already. My bad :/